_data/retail.yml

```
@@ -104,6 +119,12 @@ websites:
     url: https://www.costco.com
     img: costco.png
     facebook: Costco
+    regions:
+      - ca
+      - us
+      - gb
+      - mx
+      - es
```

```suggestion
      - es
      - jp
      - kr
      - tw
      - au
      - is
      - fr
      - cn
```
Costco also serves [these locations](https://en.wikipedia.org/wiki/Costco).